aptos-move/framework/aptos-framework/sources/genesis.move

```diff
@@ -62,7 +62,19 @@ module aptos_framework::genesis {
         aptos_governance::store_signer_cap(&aptos_framework_account, @aptos_framework, aptos_framework_signer_cap);
 
         // put reserved framework reserved accounts under aptos governance
-        let framework_reserved_addresses = vector<address>[@0x2, @0x3, @0x4, @0x5, @0x6, @0x7, @0x8, @0x9, @0x10];
+
+        // TODO: const vectors are not supported yet in the Move prover
+        let framework_reserved_addresses: vector<address> = vector::empty();
```

Comment:
This looks a bit ugly IMO but will leave it to @junkil-park and @meng-xu-cs regarding whether we can fix this on the prover side.

Reply:
@movekevin can you please look at the PR history before approving? A better fix for the core problem is on its way.

Reply:
Sorry I misread that comment as referring a different part of the change. But yes totally agree 👍